imports/components/@primitives/UI/cards/Card.js

```diff
@@ -1,214 +1,149 @@
-import { Component, PropTypes } from "react";
+
+// @flow
 import { Link } from "react-router";
+// $FlowMeteor
+import { ifElse, compose, join, flatten, append, merge, identity, concat } from "ramda";
 
 import { ImageLoader } from "../../UI/loading";
-// import { ImageLoader } from "/imports/components/loading";
-// import Styles from "../loading/FeedItemSkeleton-css";
 
-const Wrapper = (props) => (
-  <div {...props}>
-    {props.children}
-  </div>
+type IWrapper = { children: Object };
+export const Wrapper = (props: IWrapper) => <div {...props}>{props.children}</div>;
+
+export const createItemClasses = (itemClasses?: [string], linkAll: boolean): string =>
+  compose(
+    join(" "),
+    flatten,
+    ifElse(() => itemClasses, append(itemClasses), (x) => x),
+    ifElse(() => linkAll, append("background--light-primary"), (x) => x)
+  )(["card__item", "soft", "text-center", "soft-double-ends"]);
```

Comment:
True. This is pretty rough if you're not familiar with the API. Compose runs the parameters in reverse starting from the bottom and passes the results into the next function up the chain. `compose(a, b, c, d)` is equivalent to `a(b(c(d)))` so `d` technically gets evaluated first.